### DISCUSSION TITLE
Version 1.19.6: March 6, 2024 Enhancement Performance improvement. what is the detail?

### DISCUSSION THREAD

**Discussion by heartacker** [microsoft/vscode-cpptools#12075] [General]
Version 1.19.6: March 6, 2024
Enhancement
Performance improvement.

**Comment by Colengms** [comment]
Hi @heartacker .  The change was a bit esoteric and related to internal message processing.  The result is a minor performance improvement with message processing in general, preventing it from getting bottlenecked as easily, which potentially applies to most features.
